src/lazy/expanded/macro_table.rs

```diff
@@ -707,8 +938,9 @@ impl MacroTable {
         }
     }
 
+    // TODO: Remove this
```

Comment:
🗺️ Once we implement the encoding module sequence, we will no longer initialize the local macro table by cloning the system macro table.